master/buildbot/test/unit/test_www_oauth.py

```diff
@@ -130,6 +131,9 @@ def test_GithubVerifyCode(self):
                 login="bar",
                 name="foo bar",
                 email="bar@foo"),
```

Comment:
you should remove email="bar@foo" here, as this PR is about having this value to be wrong